src/main/java/caculator/Calculator.java

```diff
@@ -0,0 +1,90 @@
+package caculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class Calculator {
```

Comment:
```suggestion
public class Calculator {
```
접근제한자를 생략하면 같은 패키지 내에서만 접근 가능한 클래스가 되어버립니다.
의도한게 아니라면 다른 패키지에서 참조할 수 없게 되어버립니다.

테스트에서 해당 객체를 사용할 수 있었던 이유도 같은 calculator 패키지에 위치하고 있었기 때문입니다.
따라서 기본적으로 public을 붙여주고 상황에 따라 다른 접근제한자를 붙여주는 것이 좋을 것 같아요.